docs/rules/web-matrix.md

```diff
@@ -1,50 +1,51 @@
-| Edit DOM elements to make it invisible | 🚫 | 🚫 |  | 🚫 | 🚫 | |
-| Modify a CSS property directly | 🚫 | 🚫 |  | 🚫 | 🚫 | |
-| Using try...catch...finally calls |  | ✅ |  | ✅ | | |
-| The use of methods for basic operations |  |  |  |  | | |
-| Call a DOM element multiple times without caching | 🚫 | 🚫 | ✅ | 🚫 | 🚫 | |
-| Use global variables | ✅ |  |  |  |  | ✅ | | |
-| Using strings as arguments to SetTimeout() and setInterval() | 🚫 | 🚫 | ✅ | 🚫 | 🚫 | |
-| Using for...in loops | ✅ |  |  |  | | |
-| Rewrite native getter/setters |  |  |  | ✅ | | |
-| Unnecessarily assigning values to variables | ✅  |  |  |  | | |
-| Use single quote (') instead of quotation mark (") |  | ✅  | | | | |
-| Use the $i++ variable during an iteration | ✅  | ✅  |  |  | | |
-| Calling a function in the declaration of a for loop | ✅  | ✅  |  | ✅  | | |
-| Perform an SQL query inside a loop | ✅  | ✅  |  | ✅ | | |
-| Write SELECT * FROM | ✅  | ✅  |  | ✅  | | |
-| Calling a Spring repository inside a loop | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Getting the size of the collection in the loop | ✅ | |  |  |  |  |
-| Multiple if-else statement | ✅ |  |  |  |  |  |
-| Usage of static collections | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Usage Pattern.compile() in a non-static context | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Concatenate Strings in loop | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Const parameter in batch update | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Free resources | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Initialize builder/buffer with the appropriate size | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Optimize read file exceptions | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| Usage of preparedStatement instead of Statement | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
-| usage of system.arraycopy to copy arrays | ✅ | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | Use official social media sharing buttons | 🚫 | 🚫 | 🚧 | 🚫 | 🚫 |
+|  | Include a CSS file containing directives not used on a page | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | Non-grouped similar CSS declarations | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
+|  | CSS shorthand notations not used | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | CSS print not included | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | Non-standard fonts used | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | Non-outsourced CSS and Javascript | 🚫 | 🚫 | 🚧 | 🚫 | 🚫 |
+|  | Image tags containing an empty SRC attribute | 🚫 | 🚫 | 🚫 | 🚫 | 🚫 |
+|  | Resize images outside the browser | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
+|  | Use unoptimized vector images | 🚧 | 🚧 | 🚧 | 🚧 | 🚧 |
+|  | Using too many CSS/javascript animations | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
+|  | Modify the DOM when traversing it | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
+|  | Edit DOM elements to make it invisible | 🚫 | 🚫 | 🚧 | 🚫 | 🚫 |
+|  | Modify a CSS property directly | 🚫 | 🚫 | 🚧 | 🚫 | 🚫 |
+| EC34 | Using try...catch...finally calls | 🚧 | ✅ | 🚧 | ✅ | 🚧 |
+| EC22 | The use of methods for basic operations | 🚧 | ✅ | 🚧 | 🚧 | 🚧 |
+| ??? | Call a DOM element multiple times without caching | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
+| EC4 | Use global variables | ✅ | ✅ | 🚧 | ✅ | 🚧 |
+|  | Using strings as arguments to SetTimeout() and setInterval() | 🚫 | 🚫 | ✅ | 🚫 | 🚫 |
```

Comment:
I think this rule will not be reimplemented in ecoCode because it already exists in ESLint: https://eslint.org/docs/latest/rules/no-implied-eval